frontend/src/routes/(admin)/settings/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import { component$, Slot } from '@builder.io/qwik'
+import { WildebeestLogo } from '~/components/MastodonLogo'
+
+export default component$(() => {
+	return (
+		<div class="flex w-screen h-screen justify-center">
```

Comment:
should `h-screen` be `min-h-screen`, in case we've got extra stuff on the page?

Reply:
seems to have no effect but I'll change it anyway